src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Server/Properties/launchSettings.json

```diff
@@ -28,6 +29,7 @@
       "launchBrowser": true,
       //#if wasm
       "launchUrl": "",
```

Comment:
Is the url for wasm configurable... I'm wondering whether the assumption that the wasm app will be at the root is reasonable or not.

Reply:
out of scope.... as we discussed we can create an issue and provide some guidance in the template for how to change the root of the hosted WASM app if someone so desires.